twisted/conch/ssh/_kex.py

```diff
@@ -153,6 +198,19 @@ def getKex(kexAlgorithm):
     return _kexAlgorithms[kexAlgorithm]
 
 
+def isEllipticCurve(kexAlgorithm):
```

Comment:
I can see that you're just copying the pattern found below, with `isFixedGroup`, but that was probably not a great pattern and the implementation could just call `_IEllipticCurveExchangeKexAlgorithm.providedBy` directly.  (Optional feedback since this doesn't touch any public interfaces, so it's purely a matter of taste.)